tabular/src/autogluon/tabular/learner/abstract_learner.py

```diff
@@ -154,6 +154,8 @@ def _fit(self, X: DataFrame, X_val: DataFrame = None, scheduler_options=None, hy
         raise NotImplementedError
 
     def predict_proba(self, X: DataFrame, model=None, as_pandas=True, as_multiclass=True, inverse_transform=True):
+        if X.empty:
+            return pd.DataFrame()
```

Comment:
probably want to return an empty dataframe with the same columns as if it wasn't empty.